### PR TITLE
WebDriver conformance changes for Firefox 90

### PR DESCRIPTION
**Pull request by whimboo**
The following PR contains all the user facing changes to Marionette for the Firefox 90 release. Here a [list of all the changes](https://bugzilla.mozilla.org/buglist.cgi?j_top=OR&f1=cf_status_firefox90&o1=equals&resolution=FIXED&o2=equals&query_format=advanced&f2=cf_status_firefox90&v1=fixed&component=Marionette&v2=verified&product=Testing&list_id=15770104).

@juliandescottes can you please cross-check? Thanks.

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/90/index.html [hunk 1]
```diff
@@ -65,6 +65,27 @@ <h4 id="DOM">DOM</h4>
   <li>Support for the <code>matrix</code> protocol has been added and can now be passed into the {{domxref('Navigator.registerProtocolHandler()')}} method as a valid scheme.</li>
 </ul>
 
+<h3 id="webdriver_conformance_marionette">WebDriver conformance (Marionette)</h3>
+<ul>
+  <li>Marionette now restricts to a single active WebDriver session ({{bug(1691047)}}).</li>
+  <li>Added support for the new type of user prompts in Firefox ({{bug(1686741)}})</li>
+  <li>Window handles make use of an unique id now, and don't change for process
```
Comment (juliandescottes) [inline files/en-us/mozilla/firefox/releases/90/index.html:72]:
```suggestion
  <li>Window handles make use of a unique id now, and don't change for process
```
nit: an unique -> a unique

**Review by juliandescottes — APPROVED** [review]
lgtm

**Comment by whimboo** [comment]
Thanks Julian! I've made that update and another fix for the closing `li` tag. 

@Rumyra can you please review? Thanks.

**Review by Rumyra — APPROVED** [review]

**Comment by Rumyra** [comment]
Thnaks both! Appreciated :)

All merged 👍

**Pull request merged**